swagger.yaml

```diff
@@ -324,7 +324,119 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
-
+  /equipmentstatus/remove_from_repair/{equipmentstatusId}:
```

Comment:
I think API might be more restful like this:
- PUT /equipmentstatus/repaired/{equipmentstatusId}/ (to mark as repaired)
- DELETE /equipmentstatus/repaired/{equipmentstatusId} (to remove from repaired list)
- PATCH /equipmentstatus/repaired/{equipmentstatusId} (to update repaired equipment details)